### PR TITLE
add base values and importances for ending tokens to QA explanations

### PR DESCRIPTION
**Pull request by imatiach-msft**
## Description

Add base values and importances for ending tokens to QA explanations.

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [x] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: responsibleai_text/responsibleai_text/managers/explainer_manager.py [hunk 1]
```diff
@@ -119,13 +119,17 @@ def compute(self):
             qa_predictor = QAPredictor(self._model)
             qa_start = qa_predictor.predict_qa_start
             qa_start.__func__.output_names = qa_predictor.output_names
-            explainer = shap.Explainer(qa_start, self._model.tokenizer)
+            explainer_start = shap.Explainer(qa_start, self._model.tokenizer)
+            qa_end = qa_predictor.predict_qa_end
+            qa_end.__func__.output_names = qa_predictor.output_names
```
Comment (vinuthakaranth) [inline responsibleai_text/responsibleai_text/managers/explainer_manager.py:124]: nit: I see qa_start.__func__.output_names and qa_end.__func__.output_names has same value qa_predictor.output_names can they be combined?

File: responsibleai_text/responsibleai_text/managers/explainer_manager.py [hunk 2]
```diff
@@ -251,21 +263,25 @@ def _compute_text_feature_importances(self, explanation):
...
-                text_feature_importance.text = instance.data
-            else:
-                raise ValueError("Unknown task type: {}".format(
-                    self._task_type))
-            text_feature_importances.append(text_feature_importance)
+                    [i_start.values.tolist(), i_end.values.tolist()]
+                text_feature_importance.text = i_start.data
+                text_feature_importance.baseValues = \
+                    [i_start.base_values.tolist(), i_end.base_values.tolist()]
+                text_feature_importances.append(text_feature_importance)
```
Comment (vinuthakaranth) [inline responsibleai_text/responsibleai_text/managers/explainer_manager.py:281]: For my knowledge: why is this last append needed?

**Review by vinuthakaranth — APPROVED** [review]

File: responsibleai_text/responsibleai_text/managers/explainer_manager.py (hunk 1 above)
Comment (imatiach-msft) [reply responsibleai_text/responsibleai_text/managers/explainer_manager.py:124]: done!

File: responsibleai_text/responsibleai_text/managers/explainer_manager.py (hunk 2 above)
Comment (imatiach-msft) [reply responsibleai_text/responsibleai_text/managers/explainer_manager.py:281]: text_feature_importance contains an explanation for each document and text_feature_importances, with the extra s, contains the explanations for all documents.  So the append is called for each additional document or text phrase/sentence we are running on.

**Pull request merged**